test/db/formats/rel

```diff
@@ -0,0 +1,104 @@
+NAME=uncached err
+FILE=bins/rel/d_profileNP.rel
+ARGS=-e bin.cache=false
+CMDS=
+EXPECT_ERR=<<EOF
+INFO: REL module ID 00000001
+INFO: REL version 3
+Warning: run r2 with -e bin.cache=true to fix relocations in disassembly
+EOF
+RUN
+
+NAME=sections
+FILE=bins/rel/d_profileNP.rel
+ARGS=-e bin.cache=true -B 0x80500000
+CMDS=iS
+EXPECT=<<EOF
+[Sections]
+
+nth paddr       size vaddr       vsize perm name
+------------------------------------------------
+0   0x000000d0  0xa0 0x805000d0   0xa0 -rwx text_1
+1   0x00000170   0x4 0x80500170    0x4 -rw- data_2
+2   0x00000174   0x4 0x80500174    0x4 -rw- data_3
+3   0x00000178  0x34 0x80500178   0x34 -rw- data_4
+4   0x00000000   0x4 0x80500380    0x4 -rw- bss
+
+EOF
+RUN
+
+NAME=dump relocs
+FILE=bins/rel/d_profileNP.rel
+ARGS=-e bin.cache=true -B 0x80500000 -e log.level=5
+CMDS=
+EXPECT_ERR=<<EOF
+INFO: REL module ID 00000001
+INFO: REL version 3
+DEBUG: imp 0: 2 relocs
+DEBUG: imp 1: 2 relocs
+DEBUG: imp 2: 2 relocs
+DEBUG: imp 3: 2 relocs
+DEBUG: imp 4: 2 relocs
+DEBUG: imp 5: 2 relocs
+DEBUG: imp 6: 7 relocs
+DEBUG: imp 7: 16 relocs
+ERROR: Imports from other REL (00000002) not yet implemented
```

Comment:
this is a warning, if it was an error. it will not report anything after this message.

Reply:
lets say warning or debug or info, but not error